### ISSUE TITLE
Is there any way to inject Realm to mobile nodejs v8 environment?

### ISSUE THREAD

**Issue by GHChrisSu** [realm/realm-js#3717]
We currently do a project that running a nodejs mobile version in android and ios, and we found that in mobile can only access by react-native, if we running a nodejs in mobile how to inject Realm to nodejs environment?

I mean using the library like this https://github.com/JaneaSystems/nodejs-mobile
and still have react-native version https://github.com/JaneaSystems/nodejs-mobile-react-native
but what I want is compile realm-js to the realm.node for mobile

Currently jsc have been override nodejs using v8 maybe some way can override v8 to nodejs, I think they are all in same thread, if library load in one place

Goals
Make realm object can be access in nodejs mobile environment

Expected Results
nodejs-mobile can access realm object

Actual Results
Realm object undefined

Version of Realm and Tooling
Realm JS SDK Version: 10.4.0
Node or React Native: v12.19.0
Client OS & Version: android, ios
Which debugger for React Native: ?/None

**Comment by kneth** [comment]
First a disclaimer: We haven't tried to integrate with https://github.com/JaneaSystems/nodejs-mobile, and it is not part of our roadmap.

I have briefly looked at https://github.com/JaneaSystems/nodejs-mobile/issues/316, and it is virtually impossible to see the actual error message. I would imagine it tries to build from source, and the question is if you have the entire toolchain installed (C++ compilers for iOS and Android, CMake, cmake-js, etc.)?

**Comment by GHChrisSu** [comment]
Yes, I have installed the toolchain and I compile the sqlite3.node for android.
